main.go

```diff
@@ -3,75 +3,121 @@ package main
 import (
 	"flag"
 	"log"
-	"sync"
+
+	"github.com/Regner/albionmarket-client/utils"
 
 	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
 	"github.com/google/gopacket/pcap"
-	"github.com/regner/albionmarket-client/assemblers"
-	"github.com/regner/albionmarket-client/utils"
+	"github.com/hmadison/ao_spectator/photon"
+	"github.com/hmadison/ao_spectator/albion"
+	"github.com/mitchellh/mapstructure"
 )
 
+type RequestBuyOrders struct {
```

Comment:
Why not just use the RequestBuyOrders from the ao_spectator/albion package?

https://github.com/hmadison/ao_spectator/blob/master/albion/packets.go